### PR TITLE
Add fullstop and grammar fixes + pack metadata bumps

### PR DESCRIPTION
**Pull request by agamm**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related to: demisto/etc#41039 (This is a POC PR).
previous version: https://github.com/demisto/content/pull/14714 (without fullstops and pack bumps)

## Explanation of fixes:
QualysV2 0/641
```
The word ‘operation’ doesn’t fit in this context.
Before: Endpoint operation system
After: Endpoint operating system
This sentence does not start with an uppercase letter.
Before: start minute of the scheduled report
After: Start minute of the scheduled report
This abbreviation for “identification” is spelled all-uppercase.
Before: Appliance ids of the asset group
After: Appliance IDs of the asset group
This abbreviation for “identification” is spelled all-uppercase.
Before: Default appliance ids of the asset group
After: Default appliance IDs of the asset group
This sentence does not start with an uppercase letter.
Before: key name, either ID or REFERENCE
After: Key name, either ID or REFERENCE
```

XFE_v2 1/641
```
This sentence does not start with an uppercase letter.
Before: the title of the CVE
After: The title of the CVE
```

jamf v2 2/641
```
Possible typo: you repeated a whitespace
Before: The mobile device  initial entry date in epoch format.
After: The mobile device initial entry date in epoch format.
Possible typo: you repeated a whitespace
Before: The mobile device  initial entry date in UTC format.
After: The mobile device initial entry date in UTC format.
Two determiners in a row. Choose either “the” or “the”.
Before: Whether the the mobile device has data roaming enabled.
After: Whether the mobile device has data roaming enabled.
```

XFE_v2 1/641
```
This sentence doesn't end with a full stop.
Before: The IP address
After: The IP address.
This sentence doesn't end with a full stop.
Before: For malicious IPs, the vendor that made the decision
After: For malicious IPs, the vendor that made the decision.
This sentence doesn't end with a full stop.
Before: For malicious IPs, the reason for the vendor to make the decision
After: For malicious IPs, the reason for the vendor to make the decision.
This sentence doesn't end with a full stop.
Before: For malicious IPs, the score from the vendor
After: For malicious IPs, the score from the vendor.
This sentence doesn't end with a full stop.
Before: The reason for the given score from X-Force Exchange
After: The reason for the given score from X-Force Exchange.
This sentence doesn't end with a full stop.
Before: Additional details of the score's reason
After: Additional details of the score's reason.
This sentence doesn't end with a full stop.
Before: The subnets of the IP
After: The subnets of the IP.
This sentence doesn't end with a full stop.
Before: The given URL from the user
After: The given URL from the user.
This sentence doesn't end with a full stop.
Before: For malicious URLs, the vendor that made the decision
After: For malicious URLs, the vendor that made the decision.
This sentence doesn't end with a full stop.
Before: The file's MD5
After: The file's MD5.
This sentence doesn't end with a full stop.
Before: The file's SHA1
After: The file's SHA1.
This sentence doesn't end with a full stop.
Before: The file's SHA256
After: The file's SHA256.
This sentence doesn't end with a full stop.
Before: For malicious files, the vendor that made the decision
After: For malicious files, the vendor that made the decision.
This sentence doesn't end with a full stop.
Before: For malicious files, the reason for the vendor to make the decision
After: For malicious files, the reason for the vendor to make the decision.
This sentence doesn't end with a full stop.
Before: C&C servers related to the given file
After: C&C servers related to the given file.
This sentence doesn't end with a full stop.
Before: Emails related to the given file
After: Emails related to the given file.
This sentence doesn't end with a full stop.
Before: Download servers related to the given file
After: Download servers related to the given file.
This sentence doesn't end with a full stop.
Before: Subjects related to the given file
After: Subjects related to the given file.
This sentence doesn't end with a full stop.
Before: Additional information about the given file
After: Additional information about the given file.
This sentence doesn't end with a full stop.
Before: Domain
After: Domain.
This sentence doesn't end with a full stop.
Before: For malicious URLs, the vendor that made the decision
After: For malicious URLs, the vendor that made the decision.
This sentence doesn't end with a full stop.
Before: The ID of the CVE
After: The ID of the CVE.
This sentence doesn't end with a full stop.
Before: The CVSS of the CVE
After: The CVSS of the CVE.
This sentence doesn't end with a full stop.
Before: The date this was published
After: The date this was published.
This sentence doesn't end with a full stop.
Before: The CVE description
After: The CVE description.
This sentence doesn't end with a full stop.
Before: The XFBID of the CVE
After: The XFBID of the CVE.
This sentence doesn't end with a full stop.
Before: The risk level of the CVE
After: The risk level of the CVE.
This sentence doesn't end with a full stop.
Before: The reported date of the CVE
After: The reported date of the CVE.
This sentence doesn't end with a full stop.
Before: The CVSS information of the CVE
After: The CVSS information of the CVE.
This sentence doesn't end with a full stop.
Before: The CVE stdcodes
After: The CVE stdcodes.
This sentence doesn't end with a full stop.
Before: The title of the CVE
After: The title of the CVE.
This sentence doesn't end with a full stop.
Before: The description of the CVE
After: The description of the CVE.
This sentence doesn't end with a full stop.
Before: The affetcted platforms due to the CVE
After: The affetcted platforms due to the CVE.
This sentence doesn't end with a full stop.
Before: The date the host will be expired
After: The date the host will be expired.
This sentence doesn't end with a full stop.
Before: Contact information of the host's organization
After: Contact information of the host's organization.
This sentence doesn't end with a full stop.
Before: The total rows received after search
After: The total rows received after search.
```

QualysV2 2/641
```
This sentence doesn't end with a full stop.
Before: IP Addresses
After: IP Addresses.
This sentence doesn't end with a full stop.
Before: IP Range
After: IP Range.
This sentence doesn't end with a full stop.
Before: Report ID
After: Report ID.
This sentence doesn't end with a full stop.
Before: Report title
After: Report title.
This sentence doesn't end with a full stop.
Before: Report type
After: Report type.
This sentence doesn't end with a full stop.
Before: Date and time the report launched
After: Date and time the report launched.
This sentence doesn't end with a full stop.
Before: Report output format
After: Report output format.
This sentence doesn't end with a full stop.
Before: Report size
After: Report size.
This sentence doesn't end with a full stop.
Before: Report state status
After: Report state status.
This sentence doesn't end with a full stop.
Before: Report status message
After: Report status message.
This sentence doesn't end with a full stop.
Before: Report status percent
After: Report status percent.
This sentence doesn't end with a full stop.
Before: Report expiration datetime
After: Report expiration datetime.
This sentence doesn't end with a full stop.
Before: Scan REF
After: Scan REF.
This sentence doesn't end with a full stop.
Before: Scan type
After: Scan type.
This sentence doesn't end with a full stop.
Before: Scan title
After: Scan title.
This sentence doesn't end with a full stop.
Before: Date and time the scan launched
After: Date and time the scan launched.
This sentence doesn't end with a full stop.
Before: Scan Duration
After: Scan Duration.
This sentence doesn't end with a full stop.
Before: Scan Processing Priority
After: Scan Processing Priority.
This sentence doesn't end with a full stop.
Before: Scan Processed
After: Scan Processed.
This sentence doesn't end with a full stop.
Before: Scan status state
After: Scan status state.
This sentence doesn't end with a full stop.
Before: Scan status sub state
After: Scan status sub state.
This sentence doesn't end with a full stop.
Before: Scan Schedule
After: Scan Schedule.
This sentence doesn't end with a full stop.
Before: Scan Target
After: Scan Target.
This sentence doesn't end with a full stop.
Before: Target Asset Group Title
After: Target Asset Group Title.
This sentence doesn't end with a full stop.
Before: Scan Deafualt Flag
After: Scan Deafualt Flag.
This sentence doesn't end with a full stop.
Before: The user that created the scan
After: The user that created the scan.
This sentence doesn't end with a full stop.
Before: Scan ID
After: Scan ID.
This sentence doesn't end with a full stop.
Before: Scan ref
After: Scan ref.
This sentence doesn't end with a full stop.
Before: Scan REF
After: Scan REF.
This sentence doesn't end with a full stop.
Before: Scan type
After: Scan type.
This sentence doesn't end with a full stop.
Before: Scan title
After: Scan title.
This sentence doesn't end with a full stop.
Before: Date and time the scan launched
After: Date and time the scan launched.
This sentence doesn't end with a full stop.
Before: Scan Duration
After: Scan Duration.
This sentence doesn't end with a full stop.
Before: Scan Processing Priority
After: Scan Processing Priority.
This sentence doesn't end with a full stop.
Before: Scan Processed
After: Scan Processed.
This sentence doesn't end with a full stop.
Before: Scan status state
After: Scan status state.
This sentence doesn't end with a full stop.
Before: Scan status sub state
After: Scan status sub state.
This sentence doesn't end with a full stop.
Before: Scan Schedule
After: Scan Schedule.
This sentence doesn't end with a full stop.
Before: Scan Target
After: Scan Target.
This sentence doesn't end with a full stop.
Before: Target Asset Group Title
After: Target Asset Group Title.
This sentence doesn't end with a full stop.
Before: Scan Deafualt Flag
After: Scan Deafualt Flag.
This sentence doesn't end with a full stop.
Before: The user that created the scan
After: The user that created the scan.
This sentence doesn't end with a full stop.
Before: Scan ID
After: Scan ID.
This sentence doesn't end with a full stop.
Before: Scheduled scan active
After: Scheduled scan active.
This sentence doesn't end with a full stop.
Before: IP range requested start
After: IP range requested start.
This sentence doesn't end with a full stop.
Before: IP range requested end
After: IP range requested end.
This sentence doesn't end with a full stop.
Before: Iscanner name used in the scan
After: Iscanner name used in the scan.
This sentence doesn't end with a full stop.
Before: Frequency of usage of the scan
After: Frequency of usage of the scan.
This sentence doesn't end with a full stop.
Before: Start date of the scheduled scan in UTC format
After: Start date of the scheduled scan in UTC format.
This sentence doesn't end with a full stop.
Before: Start hour of the scheduled scan
After: Start hour of the scheduled scan.
This sentence doesn't end with a full stop.
Before: Start minute of the scheduled scan
After: Start minute of the scheduled scan.
This sentence doesn't end with a full stop.
Before: Time zone code of the time for the scheduled scan
After: Time zone code of the time for the scheduled scan.
This sentence doesn't end with a full stop.
Before: Time zone details of the time for the scheduled scan
After: Time zone details of the time for the scheduled scan.
This sentence doesn't end with a full stop.
Before: Default flag of the option profile
After: Default flag of the option profile.
This sentence doesn't end with a full stop.
Before: Title of the option profile
After: Title of the option profile.
This sentence doesn't end with a full stop.
Before: Connector UUID of EC2 instance
After: Connector UUID of EC2 instance.
This sentence doesn't end with a full stop.
Before: Endpoint of EC2 instance
After: Endpoint of EC2 instance.
This sentence doesn't end with a full stop.
Before: EC2 only classic
After: EC2 only classic.
This sentence doesn't end with a full stop.
Before: Endpoint ID
After: Endpoint ID.
This sentence doesn't end with a full stop.
Before: IP
After: IP.
This sentence doesn't end with a full stop.
Before: Host's cloud provider
After: Host's cloud provider.
This sentence doesn't end with a full stop.
Before: DNS
After: DNS.
This sentence doesn't end with a full stop.
Before: EC2 instance ID
After: EC2 instance ID.
This sentence doesn't end with a full stop.
Before: QG host ID
After: QG host ID.
This sentence doesn't end with a full stop.
Before: Cloud service of the endpoint
After: Cloud service of the endpoint.
This sentence doesn't end with a full stop.
Before: Tracking method of the endpoint
After: Tracking method of the endpoint.
This sentence doesn't end with a full stop.
Before: Cloud resource ID of the endpoint
After: Cloud resource ID of the endpoint.
This sentence doesn't end with a full stop.
Before: Domain of the endpoint
After: Domain of the endpoint.
This sentence doesn't end with a full stop.
Before: Host name of the endpoint
After: Host name of the endpoint.
This sentence doesn't end with a full stop.
Before: NETBIOS
After: NETBIOS.
This sentence doesn't end with a full stop.
Before: Endpoint operating system
After: Endpoint operating system.
This sentence doesn't end with a full stop.
Before: IP
After: IP.
This sentence doesn't end with a full stop.
Before: Port
After: Port.
This sentence doesn't end with a full stop.
Before: Fully qualified domain name
After: Fully qualified domain name.
This sentence doesn't end with a full stop.
Before: Date and time of the executed manage action
After: Date and time of the executed manage action.
This sentence doesn't end with a full stop.
Before: Result message of the executed action
After: Result message of the executed action.
This sentence doesn't end with a full stop.
Before: IP Address
After: IP Address.
This sentence doesn't end with a full stop.
Before: IP of excluded host with expiration date
After: IP of excluded host with expiration date.
This sentence doesn't end with a full stop.
Before: Expiration date of excluded host address
After: Expiration date of excluded host address.
This sentence doesn't end with a full stop.
Before: Range of excluded hosts with expiration date
After: Range of excluded hosts with expiration date.
This sentence doesn't end with a full stop.
Before: Expiration date of excluded hosts ranges
After: Expiration date of excluded hosts ranges.
This sentence doesn't end with a full stop.
Before: Range of IP addresses
After: Range of IP addresses.
This sentence doesn't end with a full stop.
Before: Report active
After: Report active.
This sentence doesn't end with a full stop.
Before: Title of the template
After: Title of the template.
This sentence doesn't end with a full stop.
Before: Start date of the scheduled report in UTC format
After: Start date of the scheduled report in UTC format.
This sentence doesn't end with a full stop.
Before: Start hour of the scheduled report
After: Start hour of the scheduled report.
This sentence doesn't end with a full stop.
Before: Start minute of the scheduled report
After: Start minute of the scheduled report.
This sentence doesn't end with a full stop.
Before: Frequency of the schduled report
After: Frequency of the schduled report.
This sentence doesn't end with a full stop.
Before: Timezone of the scheduled report
After: Timezone of the scheduled report.
This sentence doesn't end with a full stop.
Before: Timezone details of the scheduled report
After: Timezone details of the scheduled report.
This sentence doesn't end with a full stop.
Before: Report template ID
After: Report template ID.
This sentence doesn't end with a full stop.
Before: Report type
After: Report type.
This sentence doesn't end with a full stop.
Before: Report template title
After: Report template title.
This sentence doesn't end with a full stop.
Before: Last update time
After: Last update time.
This sentence doesn't end with a full stop.
Before: Report template global
After: Report template global.
This sentence doesn't end with a full stop.
Before: Report template default
After: Report template default.
This sentence doesn't end with a full stop.
Before: Last updated user login
After: Last updated user login.
This sentence doesn't end with a full stop.
Before: Last updated user login first name
After: Last updated user login first name.
This sentence doesn't end with a full stop.
Before: Last updated user login last name
After: Last updated user login last name.
This sentence doesn't end with a full stop.
Before: Type of report template
After: Type of report template.
This sentence doesn't end with a full stop.
Before: Vulnerability QID
After: Vulnerability QID.
This sentence doesn't end with a full stop.
Before: Is Vulnerability patchable
After: Is Vulnerability patchable.
This sentence doesn't end with a full stop.
Before: Severity level of the Vulnerability
After: Severity level of the Vulnerability.
This sentence doesn't end with a full stop.
Before: Consequence of the Vulnerability
After: Consequence of the Vulnerability.
This sentence doesn't end with a full stop.
Before: ID of the vendor
After: ID of the vendor.
This sentence doesn't end with a full stop.
Before: URL of the vendor
After: URL of the vendor.
This sentence doesn't end with a full stop.
Before: Date of the last service modification
After: Date of the last service modification.
This sentence doesn't end with a full stop.
Before: CVE ID
After: CVE ID.
This sentence doesn't end with a full stop.
Before: CVE URL
After: CVE URL.
This sentence doesn't end with a full stop.
Before: Published date
After: Published date.
This sentence doesn't end with a full stop.
Before: Additional info
After: Additional info.
This sentence doesn't end with a full stop.
Before: Discovery Authentication type
After: Discovery Authentication type.
This sentence doesn't end with a full stop.
Before: Is discovery remote
After: Is discovery remote.
This sentence doesn't end with a full stop.
Before: Diagnosis of vulnerability
After: Diagnosis of vulnerability.
This sentence doesn't end with a full stop.
Before: PCI flag
After: PCI flag.
This sentence doesn't end with a full stop.
Before: Product name
After: Product name.
This sentence doesn't end with a full stop.
Before: Vendor of the product
After: Vendor of the product.
This sentence doesn't end with a full stop.
Before: Type of the vulnerability
After: Type of the vulnerability.
This sentence doesn't end with a full stop.
Before: Title of the vulnerability
After: Title of the vulnerability.
This sentence doesn't end with a full stop.
Before: Solution for the vulnerability
After: Solution for the vulnerability.
This sentence doesn't end with a full stop.
Before: Category of the vulnerability
After: Category of the vulnerability.
This sentence doesn't end with a full stop.
Before: Asset Group ID
After: Asset Group ID.
This sentence doesn't end with a full stop.
Before: Asset Group title
After: Asset Group title.
This sentence doesn't end with a full stop.
Before: Asset Group owner ID
After: Asset Group owner ID.
This sentence doesn't end with a full stop.
Before: Asset Group unit ID
After: Asset Group unit ID.
This sentence doesn't end with a full stop.
Before: Asset Group network ID
After: Asset Group network ID.
This sentence doesn't end with a full stop.
Before: IP in the asset group
After: IP in the asset group.
This sentence doesn't end with a full stop.
Before: Asset Group IP range
After: Asset Group IP range.
This sentence doesn't end with a full stop.
Before: Appliance IDs of the asset group
After: Appliance IDs of the asset group.
This sentence doesn't end with a full stop.
Before: Default appliance IDs of the asset group
After: Default appliance IDs of the asset group.
This sentence doesn't end with a full stop.
Before: Scanned device DNS
After: Scanned device DNS.
This sentence doesn't end with a full stop.
Before: Scanned device instance
After: Scanned device instance.
This sentence doesn't end with a full stop.
Before: Scanned device IP address
After: Scanned device IP address.
This sentence doesn't end with a full stop.
Before: Scanned device Netbios
After: Scanned device Netbios.
This sentence doesn't end with a full stop.
Before: Qualys ID for vulnerabilities
After: Qualys ID for vulnerabilities.
This sentence doesn't end with a full stop.
Before: Scan result
After: Scan result.
This sentence doesn't end with a full stop.
Before: The user who executed the scan
After: The user who executed the scan.
This sentence doesn't end with a full stop.
Before: The company of the user who executed the scan
After: The company of the user who executed the scan.
This sentence doesn't end with a full stop.
Before: The date of the scan
After: The date of the scan.
This sentence doesn't end with a full stop.
Before: The scan title
After: The scan title.
This sentence doesn't end with a full stop.
Before: IP’s which were excluded from the scan
After: IP’s which were excluded from the scan.
This sentence doesn't end with a full stop.
Before: The duration of the scan
After: The duration of the scan.
This sentence doesn't end with a full stop.
Before: Number of hosts that are available during the scan
After: Number of hosts that are available during the scan.
This sentence doesn't end with a full stop.
Before: Total number of hosts that were submitted to scan
After: Total number of hosts that were submitted to scan.
This sentence doesn't end with a full stop.
Before: Type of the report
After: Type of the report.
This sentence doesn't end with a full stop.
Before: Scan option profile
After: Scan option profile.
This sentence doesn't end with a full stop.
Before: Status of the scan
After: Status of the scan.
This sentence doesn't end with a full stop.
Before: Key name of launched VM scan, either ID or a REFERENCE
After: Key name of launched VM scan, either ID or a REFERENCE.
This sentence doesn't end with a full stop.
Before: Value of the key
After: Value of the key.
This sentence doesn't end with a full stop.
Before: Key name, either ID or REFERENCE
After: Key name, either ID or REFERENCE.
This sentence doesn't end with a full stop.
Before: Value of either ID or REFERENCE
After: Value of either ID or REFERENCE.
This sentence doesn't end with a full stop.
Before: Action result message
After: Action result message.
This sentence doesn't end with a full stop.
Before: Date & time of the action
After: Date & time of the action.
This sentence doesn't end with a full stop.
Before: Action result message
After: Action result message.
This sentence doesn't end with a full stop.
Before: Date & time of the action
After: Date & time of the action.
This sentence doesn't end with a full stop.
Before: Result of action requested
After: Result of action requested.
This sentence doesn't end with a full stop.
Before: IPs action was made on
After: IPs action was made on.
This sentence doesn't end with a full stop.
Before: List of the restricted IPs
After: List of the restricted IPs.
This sentence doesn't end with a full stop.
Before: List of the restricted IPs
After: List of the restricted IPs.
This sentence doesn't end with a full stop.
Before: Action result message
After: Action result message.
This sentence doesn't end with a full stop.
Before: Date & time of the action
After: Date & time of the action.
This sentence doesn't end with a full stop.
Before: Status of the restricted ips feature
After: Status of the restricted ips feature.
```

### CODE REVIEW

**Review by bakatzir — APPROVED** [review]
you can use the `demsito-sdk update-release-notes -u documentation` https://github.com/demisto/demisto-sdk/tree/master/demisto_sdk/commands/update_release_notes#arguments
to add a proper rn file.
approving, will add the docs-approved after they'll be generated.

**Comment by agamm** [comment]
@bakatzir done. Who can go over the documentation changes? this is a POC commit to see if we can make to automated.

File: Packs/qualys/Integrations/Qualysv2/Qualysv2.yml [hunk 1]
```diff
@@ -349,37 +349,37 @@ script:
         in the requested list
     outputs:
     - contextPath: Qualys.SCAP.Scan.ID
-      description: Scan ID
+      description: Scan ID.
     - contextPath: Qualys.SCAP.Scan.Reference
-      description: Scan ref
+      description: Scan ref.
```
Comment (bakatzir) [inline Packs/qualys/Integrations/Qualysv2/Qualysv2.yml:354]:
```suggestion
      description: Scan reference.
```

File: Packs/qualys/Integrations/Qualysv2/Qualysv2.yml [hunk 2]
```diff
@@ -349,37 +349,37 @@ script:
...
+      description: Scan Schedule.
     - contextPath: Qualys.SCAP.Scan.Target
-      description: Scan Target
+      description: Scan Target.
     - contextPath: Qualys.SCAP.Scan.AssetGroupTitle
-      description: Target Asset Group Title
+      description: Target Asset Group Title.
     - contextPath: Qualys.SCAP.Scan.DeafualtFlag
-      description: Scan Deafualt Flag
+      description: Scan Deafualt Flag.
```
Comment (bakatzir) [inline Packs/qualys/Integrations/Qualysv2/Qualysv2.yml:380]:
```suggestion
      description: Scan default flag.
```

File: Packs/qualys/Integrations/Qualysv2/Qualysv2.yml [hunk 3]
```diff
@@ -460,33 +460,33 @@ script:
...
+      description: Scan Schedule.
     - contextPath: Qualys.Scan.TARGET
-      description: Scan Target
+      description: Scan Target.
     - contextPath: Qualys.Scan.ASSET_GROUP_TITLE
-      description: Target Asset Group Title
+      description: Target Asset Group Title.
     - contextPath: Qualys.Scan.DEFAULT_FLAG
-      description: Scan Deafualt Flag
+      description: Scan Deafualt Flag.
```
Comment (bakatzir) [inline Packs/qualys/Integrations/Qualysv2/Qualysv2.yml:487]:
```suggestion
      description: Scan default flag.
```

File: Packs/qualys/Integrations/Qualysv2/Qualysv2.yml [hunk 4]
```diff
@@ -528,63 +528,63 @@ script:
...
+      description: Scan status sub state.
     - contextPath: Qualys.Scan.TARGET
-      description: Scan Target
+      description: Scan Target.
     - contextPath: Qualys.Scan.ASSET_GROUP_TITLE
-      description: Target Asset Group Title
+      description: Target Asset Group Title.
     - contextPath: Qualys.Scan.DEFAULT_FLAG
-      description: Scan Deafualt Flag
+      description: Scan Deafualt Flag.
```
Comment (bakatzir) [inline Packs/qualys/Integrations/Qualysv2/Qualysv2.yml:555]:
```suggestion
      description: Scan default flag.
```

**Review by bakatzir — APPROVED** [review]

**Comment by agamm** [comment]
@bakatzir @yuvalbenshalom I added changes directly to READMEs too :) Please check if this is good.

**Review by bakatzir — APPROVED** [review]
nice

**Comment by bakatzir** [comment]
looks good, feel free to merge.

**Comment by agamm** [comment]
Tests fail because Qualys doesn't have tests or something similar, what do you say @yuvalbenshalom ?

**Comment by bakatzir** [comment]
`Packs/qualys/Integrations/Qualysv2/Qualysv2.yml: [CJ105] - Either QualysV2 does not have any test playbooks or that all test playbooks in this pack are currently skipped.`
no change in logic. merging.

**Pull request merged**